corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -250,9 +250,9 @@
                         <input type="text"
                             placeholder="{% trans "XPath expression" %}"
                             name="module_filter"
-                            value="{{ module.module_filter }}"/>
+                            value="{% if module.module_filter %}{{ module.module_filter }}{% endif %}"/>
```

Comment:
you're totally right, i didn't look at the docs and wanted to be sure an empty string wouldn't be placed in the `value` attribute. this was kind of a hot fix for a big that could potentially break a lot of people's apps